package.json

```diff
@@ -34,6 +34,7 @@
     "/lib",
     "index.d.ts",
     "build.js",
-    "deps"
+    "deps",
+    "bin"
```

Comment:
Does this include binaries for all OSes in every release package? I'm not sure that's the right way to go about prebuilding. I think https://www.npmjs.com/package/prebuild-install is the de facto standard way to do this sort of thing.

Reply:
This PR takes the prebuildify approach.
https://github.com/prebuild/prebuildify

Quoting the readme:

> Always use prebuildify --@mafintosh

The difference with those tools is that those are made for `.node` files, but the executables in this package are not `.node` files. They are real exe files.

Reply:
hm, alright, i'm convinced on the approach then.

two more qs:
1. This doesn't appear to build on Windows in CI—will this automatically build the binaries the old way if they're not present for the platform/arch?
2. What's the expected process for forming an npm release? Does it involve manually downloading the binaries from CI? If so I think that's something we should aim to automate, or at the very least document.

Reply:
> 1. This doesn't appear to build on Windows in CI—will this automatically build the binaries the old way if they're not present for the platform/arch?

Yes. Any other platform or arch will run the build script fully.  I was planning to add Windows support. After we merged this, we can look into that.

> 2. What's the expected process for forming an npm release? Does it involve manually downloading the binaries from CI? If so I think that's something we should aim to automate, or at the very least document.

Currently, the artifacts of the latest GitHub action run should be downloaded and placed under `bin` folder. 

If you want to make this automatic, we should use semantic-release bot. We need a `NPM_TOKEN` for that. The GitHub actions can download the artifacts automatically and release a new npm version.

Reply:
For now let's just add a note in the README about how to form a release.

Reply:
Added the instructions in 
https://github.com/electron/node-minidump/pull/42/commits/0f360d78e4dfc15e3a12390ac05074a10c4c2a12